src/pages/workspace/WorkspaceMoreFeaturesPage.tsx

```diff
@@ -95,7 +99,8 @@ function WorkspaceMoreFeaturesPage({policy, route}: WorkspaceMoreFeaturesPagePro
             icon: Illustrations.Coins,
             titleTranslationKey: 'workspace.moreFeatures.taxes.title',
             subtitleTranslationKey: 'workspace.moreFeatures.taxes.subtitle',
-            isActive: policy?.tax?.trackingEnabled ?? false,
+            isActive: (policy?.tax?.trackingEnabled ?? false) || !!policy?.connections?.quickbooksOnline.config.syncTax,
```

Comment:
could you add this value to separate const too please? `policy?.connections?.quickbooksOnline.config.syncTax`

Also you are using the !! in first case but not in another whereas both should be boolean so it might be cleaner if we use it for both (extracting this to its own variable will cover this)